src/Statiq.Web/Pipelines/Feeds.cs

```diff
@@ -139,5 +139,19 @@ public Feeds()
 
         public static bool IsFeed(IDocument document) =>
             document.ContainsKey(WebKeys.FeedRss) || document.ContainsKey(WebKeys.FeedAtom) || document.ContainsKey(WebKeys.FeedRdf);
+
+        private static bool TryGetFeedPath(IDocument feedDoc, string key, string defaultExtension, out NormalizedPath path)
```

Comment:
Nice!

Reply:
Thanks! I was pretty happy with how that turned out!